app/src/main/res/layout/fragment_nutrition_product.xml

```diff
@@ -36,7 +36,15 @@
                 android:scaleType="fitCenter"
                 app:srcCompat="@drawable/ic_add_a_photo_black_48dp" />
 
-
+            <Button
+                android:drawableLeft="@drawable/ic_add_a_photo_black_18dp"
```

Comment:
Could you use the `BorderButton` style to ensure the design of all buttons are the same? For example take a look at the button (with id change_ing_img) in `fragment_ingredients_product.xml` :
![image](https://user-images.githubusercontent.com/2354602/55683255-a714d100-58f2-11e9-868c-f024cf4aebab.png)

Reply:
Changes made :)